packages/cloud_firestore/cloud_firestore/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## [UNPUBLISHED]
 
+- **FIX**: Collection reference `==` operator.
```

Comment:
Could this be changed to the following please?

```
- **FIX**: `CollectionReferencePlatform` add "==" operator.
```

Just makes it more explicit it's on the platform interface. Thanks!

Reply:
done